framework-api-event-source/src/main/java/uk/gov/justice/services/eventsourcing/source/core/EventSource.java

```diff
@@ -13,32 +13,39 @@
     /**
      * Get a stream of events by stream id.
      *
-     * @param streamId - the stream id of the stream to be retrieved
+     * @param streamId the id of the stream to be retrieved
      * @return the {@link EventStream}
      */
     EventStream getStreamById(final UUID streamId);
 
+    /**
+     * Get a stream of event streams.
+     *
+     * @return a stream of {@link EventStream}s
+     */
+    Stream<EventStream> getStreams();
 
     /**
-     * Get a streams Of eventStreams.
+     * Get a stream of event streams.
      *
-     * @return the {@link EventStream}
+     * @param position the position in the stream of streams to start from
+     * @return a stream of {@link EventStream}s
      */
-    Stream<EventStream> getEventStreamsBySequence(final long sequenceNumber);
+    Stream<EventStream> getStreamsFrom(final long position);
 
     /**
      * Clones the stream into a new stream id.
      *
-     * @param streamId - the stream id of the stream to be cloned
-     * @return the StreamId of the cloned stream.
+     * @param streamId the stream id of the stream to be cloned
```

Comment:
would this be id of the stream to be cloned as well? (remove word stream - if keeping consistent?)